style.css

```diff
@@ -436,6 +439,11 @@ label {
   justify-content: space-around;
 }
 
+.error-message {
```

Comment:
- [ ] I think we could do better with the styling of the message. At least to increase the contrast of it because right now is hard to read.

![image](https://user-images.githubusercontent.com/79114233/141132049-b56e1f71-0d34-4c6d-8843-054e63abd240.png)

Contrast:
![image](https://user-images.githubusercontent.com/79114233/141132766-3807f636-161e-48fd-b4c4-cf7099858a75.png)